components/EditEvent/index.js

```diff
@@ -1,42 +1,51 @@
 import { useState } from "react";
 import styled from "styled-components";
 
-const StyledFormList = styled.ul`
-  list-style: none;
-  margin: 30px;
-  font-family: monospace;
-  display: grid;
-  padding: 3px;
-  width: 70%;
+const ButtonContainer = styled.div`
+  margin-top: 10px;
 `;
-
 const Input = styled.input`
+  flex: 1 0 auto;
   font-family: sans-serif;
   outline: 3px;
-  height: 35px;
-  width: 210px;
+  height: 40px;
+  width: 95%;
   border: 0px;
```

Comment:
If it's 0, it's always without a unit:

```suggestion
  border: 0;
```